src/main/java/org/example/springboot_labb2/StartUpApplicationListener/StartupApplicationListener.java

```diff
@@ -0,0 +1,23 @@
+package org.example.springboot_labb2.StartUpApplicationListener;
```

Comment:
The issue you are getting are because the package you added have uppercase. If you just refactor the package to lowercase, this will solve the issue.

Reply:
That's the weird thing, it is in lowecase on my intellij. It somehow wasn't included in the push.